retrofit/src/main/java/retrofit/ExceptionCatchingRequestBody.java

```diff
@@ -49,6 +49,10 @@
     });
   }
 
+  @Override public void close() throws IOException {
+    delegate.close();
```

Comment:
I think this might have been the StrictMode leak from the disk cache we've seen...